packages/conf-flint/conf-flint.3.0/opam

```diff
@@ -22,12 +27,14 @@ depexts: [
   ["libflint-dev"] {os-distribution = "ubuntu" }
   ["flint"] {os = "macos" & os-distribution = "homebrew"}
   ["flint"] {os-distribution = "macports" & os = "macos"}
-  ["flint" "flint-devel"] {os-distribution = "centos"}
   ["flint" "flint-devel"] {os-family = "fedora"}
   ["flint" "flint-devel"] {os-distribution = "ol"}
-  ["flint-dev"] {os-distribution = "alpine"}
+  ["flint-dev@testing"] {os-distribution = "alpine"}
   ["flint-devel"] {os-family = "opensuse"}
-  ["libflint-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["libflint-devel"] {os = "win32" &
+                         (os-distribution = "cygwinports" |
+                          os-distribution = "cygwin") }
```

Comment:
```suggestion
  ["libflint-devel"] {os = "win32" & os-distribution = "cygwinports" }
  ["libflint-devel"] {os = "cygwin" & os-distribution = "cygwin" }
```
I suggest the following instead:
- reverting the cygwinports case
- adding Cygwin support (pure Cygwin, not MinGW: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values )
- not supporting MinGW under Cygwin (which we can't because it lacks the needed system package - and which fails in CI anyway)